vignettes/rng.Rmd

```diff
@@ -321,6 +321,8 @@ plain_output(readLines(file.path(path_pkg, "NAMESPACE")))
 
 Finally, run `cpp11::cpp_register()` before compiling your package so that the relevant interfaces are created (`R/cpp11.R` and `cpp11/cpp11.cpp`).  A similar process would likely work with Rcpp without any dependency on cpp11.
 
+More interesting use with persistent streams is described in `vignette("rng_package.Rmd")`
```

Comment:
Not sure how clear 'persistent streams' is on its own. I think the use in the vignette below might be described (based on similar FFI packages) as 'using the dust RNG from C++', whereas another (this?) vignette might be 'using the dust RNG from R'